api/installation_service.go

```diff
@@ -0,0 +1,53 @@
+package api
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+)
+
+type InstallationService struct {
+	client   httpClient
+	progress progress
+}
+
+func NewInstallationService(client httpClient, progress progress) InstallationService {
+	return InstallationService{
+		client:   client,
+		progress: progress,
+	}
+}
+
+func (is InstallationService) Export(outputFile string) error {
```

Comment:
I personally don't see the gain in returning a struct if it's always going to be empty, simply to follow an "established pattern". I see your point about wanting to add/remove input/output arguments later, but I think if we ever need to add them then we can create a struct then. I think there's a balance to be struck between making something future proof and anticipating functionality that may not be there. 

Different services can have different interfaces, I guess that's the nature of having different commands/tools that do different things. If every interface looked the same it wouldn't serve much of a purpose besides simple. symmetry.